### PR TITLE
fixed incorrect message about required cmake 3.7.1, when in fact is required cmake 3.7.2

### PR DESCRIPTION
**Pull request by kingofthebongo2008**
When one opens the HLSL console and has installed cmake 3.7.1 he gets a message that cmake version 3.7.1 is required, when in fact 3.7.2 is required

### CODE REVIEW

**Comment by dmpots** [comment]
These commits do not seem to be related, can you please split them into separate pull requests.

File: tools/clang/tools/dxc/dxc.cpp [hunk 1]
```diff
@@ -198,16 +198,23 @@ int DxcContext::ActOnBlob(IDxcBlob *pBlob, IDxcBlob *pDebugBlob, LPCWSTR pDebugB
...
+  if (!needDisassembly)
     return retVal;
 
-  CComPtr<IDxcCompiler> pCompiler;
-  IFT(m_dxcSupport.CreateInstance(CLSID_DxcCompiler, &pCompiler));
-
   CComPtr<IDxcBlobEncoding> pDisassembleResult;
-  IFT(pCompiler->Disassemble(pBlob, &pDisassembleResult));
+
+  if (!m_Opts.IsRootSignatureProfile()) {
```
Comment (dmpots) [inline tools/clang/tools/dxc/dxc.cpp]: Let's use a positive test here (e.g. `if (m_Opts.IsRootSignatureProfile())`)

File: tools/clang/tools/dxc/dxc.cpp [hunk 2]
```diff
@@ -198,16 +198,23 @@ int DxcContext::ActOnBlob(IDxcBlob *pBlob, IDxcBlob *pDebugBlob, LPCWSTR pDebugB
...
+      CComPtr<IDxcCompiler> pCompiler;
+      IFT(m_dxcSupport.CreateInstance(CLSID_DxcCompiler, &pCompiler));
+      IFT(pCompiler->Disassemble(pBlob, &pDisassembleResult));
+  } else {
+      //make the same behavior as fxc, people may want to embed the root signatures in their code bases.
```
Comment (dmpots) [inline tools/clang/tools/dxc/dxc.cpp]: Wording, change make -> keep

**Comment by kingofthebongo2008** [comment]
ok. i have fixed the issues.
try now, please

**Comment by dmpots** [comment]
Thanks, but I do not see any update to the pull request. Can you create a new one for your `spill-out-headr...` branch. And then update this pull request to only contain the cmake message fix?

**Comment by kingofthebongo2008** [comment]
I have managed to create a new pull request, however i have issues updating this one. I am rather new to this kind of github usage. I have created a branch for the cmake fix, but i cannot see it in the pull requests somehow. 
dxc

**Comment by dmpots** [comment]
It looks like the problem is that the cmake branch is at a point that already has the other commit. You made this pull request based on your master branch. You can tell because it says:

    Add more commits by pushing to the master branch on kingofthebongo2008/DirectXShaderCompiler.

Usually you want to create a new branch for each pull request you create. Probably the easiest option at this point is to create a new branch with just the cmake fix and make a pull request from that. You can do that by creating the branch based on your fix commit

    git checkout -b cmake_message_fix d8fdccd05989cd9f5bcc547d5a8ace667aa24acf
    git push --set-upstream origin cmake_message_fix 

Then you can create a new pull request from the `cmake_message_fix` branch and we will abandon this one.

Thanks so much for your contributions!

**Comment by dmpots** [comment]
Split into #380 and #381.

**Pull request closed without merging**